### PR TITLE
Feat/272 syntax highlighter from explorer to ui-toolkit and feat/216 added light theme and feat/283 add to token

### LINKED ISSUE #216: Light theme for json syntax highlighter
The current theme is only set up for dark themes and looks janky when the app is in light mode

### LINKED ISSUE #272: Add syntax highlighter component to ui-toolkit
Both explorer and token apps require syntax highlighting for JSON data. Lets have a ui-toolkit component for this to share component logic and theme setup.

- [syntax highlighter in explorer](https://github.com/vegaprotocol/frontend-monorepo/blob/e26caef36889a8b1fe79d2a3bca448699100149f/apps/explorer/src/app/components/syntax-highlighter/index.tsx)
- [syntax highlighter in token app](https://github.com/vegaprotocol/frontend-monorepo/blob/af43defd514f4b53f5a228968dceb2aa633623b9/apps/token/src/components/syntax-highlighter/syntax-highlighter.tsx)

### LINKED ISSUE #283: Add syntax highlighter from ui-toolkit to new token monorepo app
Token was ported from the TFE project, where it had its own syntax highlighter.

### PR DESCRIPTION
**Pull request by sam-keen**
Closes #272 
Closes #216 
Closes #283

### CODE REVIEW

File: libs/ui-toolkit/src/components/syntax-highlighter/syntax-highlighter.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  title: 'Syntax Highlighter',
+} as Meta;
+
+const data = {
+  name: 'Level one',
+  level: 1,
+  nestedData: {
+    name: 'Level two',
+    level: 2,
+  },
```
[the remark this answers is not in the document]
Comment (dexturr) [reply libs/ui-toolkit/src/components/syntax-highlighter/syntax-highlighter.stories.tsx:15]: Just to show off different colours 👍

[the remark this answers is not in the document]
Comment (sam-keen) [reply libs/ui-toolkit/src/components/syntax-highlighter/syntax-highlighter.stories.tsx:15]: Nice, thanks Dex

File: libs/ui-toolkit/src/components/syntax-highlighter/syntax-highlighter.tsx [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import Highlighter from 'react-syntax-highlighter';
+
+export const SyntaxHighlighter = ({ data }: { data: unknown }) => {
+  return (
+    <div className="sh-wrapper">
```
Comment (dexturr) [inline libs/ui-toolkit/src/components/syntax-highlighter/syntax-highlighter.tsx:5]: chore (blocking): shortened name isn't very readable, can we hvae 
```suggestion
    <div className="syntax-highlighter-wrapper">
```

**Review by edd — APPROVED** [review]

**Review by mattrussell36 — APPROVED** [review]
LGTM. Although lets try not to make unrelated changes if we can makes for a harder review 🕊️

**Pull request merged**